parser/ast/ddl.go

```diff
@@ -4038,6 +4038,42 @@ func (n *RecoverTableStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// FlashBackClusterStmt is a statement to restore the cluster to the specified timestamp
+type FlashBackClusterStmt struct {
+	ddlNode
+
+	AsOf *AsOfClause
```

Comment:
If the `AsOfClause` is required, why not making it a value instead of a pointer?

Reply:
+1

Reply:
updated, PTAL @xhebox @tangenta